CONTRIBUTING.md

```diff
@@ -78,6 +78,46 @@ Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe
 
 * Discord - [#jest](https://discordapp.com/channels/102860784329052160/103622435865104384) on [Reactiflux](http://www.reactiflux.com/)
 
+## How to try a development build of Jest in another project
```

Comment:
Am glad to move it, because indeed a principle of writing not leave and return to a topic.

What concerned me was putting info that seems of interest to a few before info that is important to most: CLA, Bugs, and How to Get in Touch.

By the way, there are some inconsistencies in h2 versus h3 level of heading, like under **Bugs** do you want me to fix those when I make requested changes?